### PR TITLE
front: add times and stops e2e test for operational studies

### LINKED ISSUE #8646: front: add e2e tests operational studies - times & stops
### Description and goal

Implement an e2e test to cover https://github.com/osrd-project/osrd-tests/issues/59

### PR DESCRIPTION
**Pull request by Maymanaf**
Closes [#8646](https://github.com/OpenRailAssociation/osrd/issues/8646)

### CODE REVIEW

**Comment by emersion** [comment]
This component will be re-designed and rewritten soon. I wonder, is now a good time to merge a test, given we're going to have to rewrite it soon?

**Review by Uriel-Sautron — COMMENTED** [review]
Thank you for this test, but it sometimes fails locally and is flaky in the CI

**Comment by kmer2016** [comment]
> This component will be re-designed and rewritten soon. I wonder, is now a good time to merge a test, given we're going to have to rewrite it soon?

After talking with @clarani, we both think it still makes sense to merge these tests. Since the structure will mostly change but the content will stay the same, this will help standardize the input table and make it easier when we update it later.

**Review by kmer2016 — APPROVED** [review]

File: front/tests/pages/op-timetable-page-model.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ class OperationalStudiesTimetablePage {
 
   readonly spaceTimeChart: Locator;
 
+  readonly speedSapceChart: Locator;
```
Comment (Uriel-Sautron) [inline front/tests/pages/op-timetable-page-model.ts]:
```suggestion
  readonly speedSpaceChart: Locator;
```
Typo

**Review by Uriel-Sautron — APPROVED** [review]
LGTM, tested

File: front/tests/pages/op-input-table-page-model.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+    const normalizedHeaderName = cleanWhitespace(header);
+
+    const headersCount = await this.columnHeaders.count();
+    let columnIndex = -1;
+
+    for (let headerIndex = 0; headerIndex < headersCount; headerIndex += 1) {
+      const headerText = await this.columnHeaders.nth(headerIndex).innerText();
+      const normalizedHeaderText = cleanWhitespace(headerText);
+      if (normalizedHeaderText === normalizedHeaderName) {
```
Comment (anisometropie) [inline front/tests/pages/op-input-table-page-model.ts]: normalizedHeaderText / normalizedHeaderName
a bit hard to follow, you could use more explicit names.

headerText   = currentColumnHeader ?
normalizedheaderName = columnHeader

**Review by anisometropie — CHANGES_REQUESTED** [review]

File: front/tests/pages/op-input-table-page-model.ts (hunk 2 above)
Comment (Maymanaf) [reply front/tests/pages/op-input-table-page-model.ts]: changed to : 
headerText = columnHeader
normalizedHeaderText = currentColumnHeader
normalizedHeaderName = expectedColumnHeader

**Review by anisometropie — APPROVED** [review]
